recipes/utm_latlong_convertor/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class UTMLatLongConvertorConan(ConanFile):
+    name = "ofxgeo"
+    description = "A convertor for Latitude Longitude to UTM Co-ordinate and vice versa."
+    topics = ("conan", "utm_latlong_convertor", "utm", "latitude", "longitude")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bakercp/ofxGeo"
+    license = "MIT"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE.md", dst="licenses",
+                  src=self._source_subfolder)
+
+        self.copy(pattern="UTM.h", dst="include",
+                  src=os.path.join(self._source_subfolder, "libs", "UTM", "include", "UTM"))
+
+        tools.replace_in_file(os.path.join(self.package_folder, "include", "UTM.h"),
+                              "#include \"ofConstants.h\"", "// #include \"ofConstants.h\"")
+        tools.replace_in_file(os.path.join(self.package_folder, "include", "UTM.h"), "#define UTM_EP2		(UTM_E2/(1-UTM_E2))	///< e'^2", """#define UTM_EP2		(UTM_E2/(1-UTM_E2))	///< e'^2
+#define DEG_TO_RAD (M_PI/180.0)
+#define RAD_TO_DEG (180.0/M_PI) """,)
```

Comment:
Shouldn't you build a lib? There are others public headers in this library, and .cpp files.

Reply:
So, the functionality to convert from UTM to Lat Lon is just this one file called `UTM.h`. Rest of the files have other functionalities which are non overlapping. So I thought of making it only a `header-only` with that one file. The `replace_in_file` is just a way around that  dependancy on `ofConstants.h`.

It would be really helpful if you have a better and a cleaner solution in mind. Thanks :)

Reply:
2 solutions:
1. create a recipe for openFrameworks first, then create your recipe
2. create your recipe as is, but add a TODO for full implementation of `ofxgeo` (which is not only this header file), and display a warning for consumers that this recipe is just a subset of `ofxgeo`.